src/grafana/chart/templates/uds-package.yaml

```diff
@@ -28,23 +28,31 @@ spec:
         targetPort: 3000
 
     allow:
-      - direction: Ingress
+      # Egress allowed to Loki
+      - direction: Egress
         selector:
           app.kubernetes.io/name: grafana
-        remoteNamespace: tempo
+        remoteNamespace: loki
         remoteSelector:
-          app.kubernetes.io/name: tempo
-        port: 9090
-        description: "Tempo Datasource"
+          app.kubernetes.io/name: loki
+        description: "Loki Datasource"
+        port: 8080
 
+      # Egress allowed to Prometheus
       - direction: Egress
         selector:
           app.kubernetes.io/name: grafana
-        remoteGenerated: Anywhere
+        remoteNamespace: monitoring
+        remoteSelector:
+          app.kubernetes.io/name: prometheus
+        description: "Prometheus Datasource"
+        port: 9090
 
+      # Egress allowed to Keyclaok
       - direction: Egress
-        remoteNamespace: tempo
+        selector:
+          app.kubernetes.io/name: grafana
+        remoteNamespace: keycloak
         remoteSelector:
-          app.kubernetes.io/name: tempo
-        port: 9411
-        description: "Tempo"
+          app.kubernetes.io/name: keycloak
+        description: "Keycloak Datasource"
```

Comment:
very nit - technically keycloak isn't a grafana datasource so this could be SSO Provider / token?

Reply:
```suggestion
        description: "SSO Provider"
```